src/lib/pages/backoffice/Acquisition/Order/OrderDetails/OrderDetails.js

```diff
@@ -146,6 +147,12 @@ class OrderPanels extends React.Component {
         <Segment attached className="bo-metadata-segment">
           <OrderInformation order={data} />
         </Segment>
+        <Header as="h3" attached="top">
+          Literature request
+        </Header>
+        <Segment attached className="bo-metadata-segment">
+          <OrderDocumentRequest orderId={data} />
```

Comment:
Why not simply adding this in the metadata on the top "Order information", similar to `Provider`, to keep it compacted instead of adding yet another extra panel?